circle.yml

```diff
@@ -6,15 +6,23 @@ general:
       - production
 machine:
   post:
-    - npm install jscs@1.13.1 -g
-    - npm install mocha@2.4.5 -g
+    - npm install jscs@1.13.1 mocha@2.4.5 -g
+    - cd api && npm run test:e2e website install
```

Comment:
I'm not sure I like `e2e` it's cryptic and not very self explanatory. We use `frontend` in the `website` repo. How do you feel about that wording instead? (or any other proposition?)